_data/authors.yml

```diff
@@ -1,16 +1,11 @@
 # /_data/authors.yml
 
 Your_Name:
```

Comment:
Keep the sample and add your name below it. Don't delete it. Replace Your_name as required and use this in the post Ankita_Sareen as you have done below.